dawn/renderer/components/peripherals/BooleanSensor.js

```diff
@@ -1,51 +1,45 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import _ from 'lodash';
 import { PeripheralTypes } from '../../constants/Constants';
 
 /**
- * A component representing a boolean sensor,
- * for example a LimitSwitch.
+ * Boolean Sensor Component
  */
 class BooleanSensor extends React.Component {
```

Comment:
Since this has no state, can you convert this to a functional-style component, and convert `formatBoolean` to a helper function?